nixarr/lib/api-keys.nix

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Hmm, this is a lot better, but ideally each service wouldn't either have all API keys or none. It should be so that each api-key is owned by something like root:sonarr-api. But maybe it's not too imporant.

Reply:
Otherwise I really like the structure of this, great job!